src/controllers/update/update.manage.trusts.tell.us.about.the.former.bo.controller.ts

```diff
@@ -48,7 +56,7 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     // check for form validation errors
     const errorList = validationResult(req);
     if (!errorList.isEmpty()) {
-      const pageProperties = getPageProperties(trust, formData, formatValidationError(errorList.array()));
+      const pageProperties = getPageProperties(trust, false, formData, formatValidationError(errorList.array()));
```

Comment:
This _wouldn't_ always be false right? You can get validation errors no-matter where you've come from. As suggested below, when basing the back-link off the `trust` object itself in every scenario, this argument could just be removed.

Reply:
Fixed